test/tests/ver-1jb/ver-1jb.i

```diff
@@ -0,0 +1,369 @@
+# Verification Problem #1jb from TMAP7 V&V document
+# Radioactive Decay of Tritium in a Distributed Trap
+
+# Physical Constants
+ideal_gas_constant = ${units 8.31446261815324 J/K/mol} # from PhysicalConstants.h
+boltzmann_constant = ${units 1.380649e-23 J/K } # from PhysicalConstants.h
+eV_to_J = 1.602176634e-19 # J/eV - from PhysicalConstants.h
+boltzmann_constant_eV = ${fparse boltzmann_constant / eV_to_J} # eV/K
```

Comment:
This can all be in one line, and we don't need to mention the units in a comment since the units system is self documenting.
```suggestion
boltzmann_constant = ${units 1.380649e-23 J/K -> eV/K } # from PhysicalConstants.h
```